src/views/contact.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends layout %}
+
+{% block content %}
+<h2>Contact</h2>
+
+<p>Want to chat with us and other members? Here are the best ways to find the group</p>
+
+<ul>
+  <li>IRC: #phpsurrey @ irc.freenode.net</li>
+  <li>Mailing list: <a href="http://eepurl.com/btyKP9">phpsurrey</a></li>
+  <li>Twitter: <a href="https://twitter.com/phpsurrey">@phpsurrey</a></li>
+  <li>Github: <a href="https://github.com/phpsurrey">phpsurrey</a></li>
+</ul>
+
+<p>We also have a <a href="http://eepurl.com/btyKP9">mailing list</a> which is super handy</p>
```

Comment:
We also have?

Maybe just put the subscription thing (from the home page) here and remove it from Ln10?